### PR TITLE
RSpec to replicate duplicate JudgeAssignTasks problem

### PR DESCRIPTION
**Pull request by yoomlam**
Bumps https://github.com/department-of-veterans-affairs/dsva-vacols/issues/174

### Description
Add RSpec to replicate duplicate JudgeAssignTasks problem.

PR #16072 should address the problem.

### Acceptance Criteria
- [ ] Code compiles correctly

### Testing Plan
Uncomment `binding.pry` in the RSpec and examine the browser tabs and task tree.
Check that json file contains no PII.

### CODE REVIEW

**Review by jcq — APPROVED** [review]
This seems like a helpful test for showing current behavior (which allows multiple active `JudgeAssignTasks`, but shouldn't); this should help with verifying that #16072 works as intended.

**Comment by eileen-nava** [comment]
Yoom, I looked at the task tree after activating the breakpoint on line 68, and I’m confused.  I see one `JudgeAssignTask` with an open status, `in_progress`.  The other `JudgeAssignTask` instances all have not-open statuses,  cancelled or completed.  Screenshot below:  
Screen Shot 2021-06-21 at 5 29 47 PM

Should I examine the task tree at a different point in the test’s progress?  Thanks!

**Comment by yoomlam** [comment]
> Yoom, I looked at the task tree after activating the breakpoint on line 68, and I’m confused. I see one `JudgeAssignTask` with an open status, `in_progress`. The other `JudgeAssignTask` instances all have not-open statuses, cancelled or completed. Screenshot below:
> ...
> 
> Should I examine the task tree at a different point in the test’s progress? Thanks!

@eileen-nava Note the `explain` page was loaded before any modifications were made to the appeal to show the imported appeal. If you reload the `explain` page in your browser, it shows the most recent task tree. Or you can `appeal.reload.treee` in the pry prompt.

**Review by eileen-nava — COMMENTED** [review]
> @eileen-nava Note the `explain` page was loaded before any modifications were made to the appeal to show the imported appeal. If you reload the `explain` page in your browser, it shows the most recent task tree. Or you can `appeal.reload.treee` in the pry prompt.

Got it, thanks! 👍

File: spec/records/appeal-121304-dup_jatasks.json [hunk 1]
```diff
@@ -0,0 +1,1302 @@
...
+    {
+      "id": 948077,
+      "file_number": "000765067",
+      "participant_id": "55927883",
+      "first_name": "Chasity",
+      "last_name": "Steuber",
+      "middle_name": "P",
+      "name_suffix": null,
+      "closest_regional_office": null,
+      "ssn": "000765067",
```
[the remark this answers is not in the document]
Comment (yoomlam) [reply spec/records/appeal-121304-dup_jatasks.json:45]: See https://secure.ssa.gov/poms.nsf/lnx/0110201035 for reference.

File: spec/fixes/investigate_duplicate_judge_assign_task_spec.rb [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+require "helpers/sanitized_json_exporter.rb"
+require "helpers/intake_renderer.rb"
+
+feature "duplicate JudgeAssignTask investigation" do
+  before do
+    User.authenticate!(css_id: "PETERSBVAM")
+    Functions.grant!("System Admin", users: ["PETERSBVAM"]) # enable access to `export` endpoint
+  end
+
+  # Ticket: https://github.com/department-of-veterans-affairs/dsva-vacols/issues/174
```
[the remark this answers is not in the document]
Comment (yoomlam) [reply spec/fixes/investigate_duplicate_judge_assign_task_spec.rb:16]: Not sure what you mean by bloat. The URL should always work. I like to include links to cross-references. In this case, the ticket provide more context in understanding the purpose of the RSpec and there may be updates to that ticket that aren't captured in this RSpec.

File: spec/records/appeal-121304-dup_jatasks.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (eileen-nava) [reply spec/records/appeal-121304-dup_jatasks.json:45]: Got it, thanks. 👍

File: spec/fixes/investigate_duplicate_judge_assign_task_spec.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (eileen-nava) [reply spec/fixes/investigate_duplicate_judge_assign_task_spec.rb:16]: Okay, fine by me.

**Pull request merged**